README.md

```diff
@@ -2086,6 +2086,35 @@ params do
 end
 ```
 
+## Parameter Validation and Coercion using `dry-schema`
+
+As an alternative to the `params` DSL described above, you can use a
+schema or `dry-validation` contract to describe an endpoint's
```

Comment:
Don't wrap lines, https://code.dblock.org/2021/06/07/to-wrap-or-not-to-wrap-in-markdown.html

Reply:
Okay, if that's your preference.